content/en/service_management/workflows/datastore.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Apps Datastore
+disable_toc: false
+further_reading:
+- link: "service_management/app_builder/build"
+  tag: "Documentation"
+  text: "Build Apps"
+- link: "service_management/workflows/build"
+  tag: "Documentation"
+  text: "Build Workflows"
+---
+
+{{< callout url="https://docs.google.com/forms/d/1gVVWLg5P3EXkVXBc4qZtqKp_nqykDsFXCMmh6pbvxx0/edit" btn_hidden="false" header="Try the Preview!">}}
+Datastore is in Preview. Use this form to request access today.
+{{< /callout >}}
+
+## Overview
+
+The Apps Datastore offers a scalable, structured data storage solution within Datadog's App Builder and Workflow Automation products. It supports CRUD (Create, Read, Update, and Delete) operations and integrates seamlessly with Datadog's ecosystem to optimize persistent data storage without the need for external databases.
+
+## Create a datastore
```

Comment:
I see that you listed here:
1. Create a datastore
2. Edit a datastore
3. Reference a datastore

The only thing that is missing here for me is "Deleting a Datastore"

Reply:
Added a delete section